### PR TITLE
Extract go sdk configuration

### PR DESCRIPTION
**Pull request by vladbarosan**
- extract Go sdk configuration

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
Fixes https://github.com/Azure/azure-rest-api-specs/issues/3442

File: specification/cognitiveservices/data-plane/AutoSuggest/readme.go.md
Comment (jhendrixMSFT) [inline specification/cognitiveservices/data-plane/AutoSuggest/readme.go.md]: Nit extra new-line.

File: specification/cognitiveservices/data-plane/ComputerVision/readme.go.md
Comment (jhendrixMSFT) [inline specification/cognitiveservices/data-plane/ComputerVision/readme.go.md]: Nit: extra new line.

**Comment by vladbarosan** [comment]
DO NOT MERGE>Will merge after sdk generation confirms no impact

File: specification/notificationhubs/resource-manager/readme.go.md
Comment (jhendrixMSFT) [inline specification/notificationhubs/resource-manager/readme.go.md]: Nit: extra new line.

File: specification/web/resource-manager/readme.go.md
Comment (jhendrixMSFT) [inline specification/web/resource-manager/readme.go.md]: Nit: extra new line.

**Review by jhendrixMSFT — COMMENTED** [review]
Found a handful of extra new-line characters at the beginning of some Go config files.

**Comment by vladbarosan** [comment]
ok should be good to go now

**Review by jhendrixMSFT — APPROVED** [review]

File: specification/advisor/resource-manager/readme.go.md
Comment (fearthecowboy) [inline specification/advisor/resource-manager/readme.go.md]: FYI -- in the `readme.go.md` having `$(go)`  guards is optional, since the file will only get loaded when --go is specified anyway. 

It's not harmful, it's just not necessary either.

**Pull request merged**
